src/Resources/Resources/help/Get-AzPrivateLinkAssociation.md

```diff
@@ -0,0 +1,163 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Resources.dll-Help.xml
+Module Name: Az.Resources
+online version: https://docs.microsoft.com/powershell/module/az.resources/get-azprivatelinkassociation/
+schema: 2.0.0
+---
+
+# Get-AzPrivateLinkAssociation
+
+## SYNOPSIS
+Gets all the Azure Resource Management Private Link Association(s).
+
+## SYNTAX
+
+```
+Get-AzPrivateLinkAssociation [-ManagementGroupId] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
```

Comment:
The syntax differs from the design in codes. Please update MD files after code changes.